content/docs/kubernetes/create-a-cluster.md

```diff
@@ -49,28 +49,30 @@ From your Civo dashboard, navigate to the [Kubernetes cluster creation page](htt
 
 The numbered sections give you options for the cluster details, complete the required information to create your cluster.
```

Comment:
I would say to make this "The sections on this page give you options for the cluster you are about to create. Complete the required fields to create your cluster."